circle.yml

```diff
@@ -1208,7 +1218,7 @@ jobs:
       <<: *defaultsParameters
       resource_class:
         type: string
-        default: medium+
+        default: large
```

Comment:
@flotwig any concerns with this? Snapshot generation is unfortunately resource intensive and I saw some OOM crashes occasionally on medium+